docs/conf/prague/2019/news/events-activities.rst

```diff
@@ -0,0 +1,62 @@
+
+Events and activities
+---------------------
+
+As always, Write the Docs events are more than just talks. Every year we offer a wide range of activities that give our community members different ways to socialize, collaborate, and connect.
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 14th**. If you're in town early, you're invited to join us for a 2-hour cruise on the Vltava river, with light snacks and drinks included. This activity is priced separately and isn't included with the conference ticket, so you're welcome to register your friends and family who might be traveling with you. You can find all the details in our `Boat Tour <https://www.writethedocs.org/conf/prague/2019/outing/>`_ page, or register directly on our `ticket website <https://ti.to/writethedocs/write-the-docs-prague-2019>`_.
+
+- **Writing Day and Welcome Reception, Sunday September 15th**. Before the main conference days, we'll be hosting a `Writing Day <https://www.writethedocs.org/conf/prague/2019/writing-day/>`_  and Welcome Reception before we dive into the main sessions. This is an excellent opportunity to register early, meet your fellow documentarians, learn about and work on interesting documentation projects, and make yourself at home!
+
+- **Lightning Talks.** One of our favorite traditions, `lightning talks <https://www.writethedocs.org/conf/prague/2019/lightning-talks/>`_ are as entertaining as they are informative. Each block of lightning talks will have 5-6 spots of up to 5 minutes each, both blocks will be after lunch during the main conference days. You can sign up on the mornings of the main conference days and prepare for some post-lunch fun.
+
+- **Unconference.** Parallel to the talks, you're welcome to join the `unconference sessions <https://www.writethedocs.org/conf/prague/2019/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. You can signup for the unconference sessions during the Writing Day and will continue during the conference.
+
+- **Social Event.** This year our social event will be held at `RKO (Restaurant by Retro), Francouzská 4, Prague 2 <https://goo.gl/maps/qJUvEuShp8kUC7ac8>`_, a bar with a great vibe owned by Retro Music Hall. The bar is about 15-20 minutes walk from the main conference venue, and is extremely well-connected by metro and tram service (Namesti Miru stop). The event will start at 20:00, and light snacks and drinks will be provided while our tab lasts. If you want a sit-down dinner we recommend to grab something on the way before we kick off the evening. **Entrance is only for conference attendees**. We will provide you with wristbands when you check-in to the conference, but your badge should work as well.
```

Comment:
Not a huge fan of wristbands. I think they are just a proxy for badges, and badges are better since they have your name on it. Is there a reason we're changing this?

Reply:
We did this as an experiment last year to accommodate for +1s and also so that people don't have to wear their badge during the party, since their name is fully visible on the badge.

Reply:
The point is to give people a choice if they want to wear their badge or wristband, some people are concerned about privacy displaying their name outside the conference venue, and also not have to worry about +1s not having a badge.

Reply:
Removed the specific text for now, we can work it out later ;)